src/models/UserModel.ts

```diff
@@ -1,32 +1,56 @@
 import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
-import Post from './PostModel';
+import { PrivateProfile, PublicProfile, Uuid } from '../types'
+import { PostModel } from './PostModel';
 import UserSession from './UserSessionModel';
 
 @Entity()
-export default class User {
+export class UserModel {
 
   @PrimaryGeneratedColumn('uuid')
-  id: string;
+  id: Uuid;
```

Comment:
why is it Uuid? and not uuid? also what does this mean, why cant it be something like userId